### PR TITLE
Enable jdk_beans on all linux

### PR DESCRIPTION
**Pull request by annaibm**
- Enable jdk_beans on alinux, plinux, zlinux for all java versions

related: eclipse-openj9/openj9#20531

### CODE REVIEW

**Comment by annaibm** [comment]
Grinder links:
slinux
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/45622/
plinux
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/45628/
alinux 
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/45634/ 
mac:
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/45685/

**Comment by llxia** [comment]
typo slinux -> zlinux

File: openjdk/playlist.xml [hunk 1]
```diff
@@ -757,7 +757,7 @@
 			<disable>
 				<comment>https://github.com/eclipse-openj9/openj9/issues/20531</comment>
 				<impl>openj9</impl>
-				<platform>^((?!(x86-64_linux)).)*$</platform>
+				<platform>^(?!.*linux).*$</platform>
 			</disable>
```
Comment (llxia) [inline openjdk/playlist.xml:761]: To enable all linux, please remove L757 - L761

**Comment by llxia** [comment]
In this case, the Adoptum master repo and branch and `disabled.jdk_beans` were used.

image

We are expecting Grinders to test PR.

**Comment by annaibm** [comment]
Grinder links:
zlinux
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/45731/ 
plinux
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/45740/  
alinux
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/45750/ 
alinux jdk8
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/45756/

**Comment by llxia** [comment]
re https://github.com/adoptium/aqa-tests/pull/5808#issuecomment-2529228842, please update PR description and commit message

File: openjdk/playlist.xml (hunk 1 above)
Comment (llxia) [reply openjdk/playlist.xml:761]: subtests are excluded in ProblemList.

**Review by llxia — APPROVED** [review]
LGTM

**Review by LongyuZhang — APPROVED** [review]

**Pull request merged**